composer.json

```diff
@@ -46,7 +46,7 @@
         "laminas/laminas-authentication": "^2.18",
         "laminas/laminas-coding-standard": "~3.0.1",
         "laminas/laminas-feed": "^2.23",
-        "laminas/laminas-filter": "^2.40",
+        "laminas/laminas-filter": "^3.0.x-dev",
```

Comment:
As mentioned by @gsteel, need this to be `^3.0`

Reply:
Thanks @Ocramius - There's a vote going on in the TSC channel on slack to release filter 👍